libhoney/transmission.py

```diff
@@ -274,7 +274,7 @@ def __init__(self, max_concurrent_batches=10, block_on_send=False,
             self.max_batch_size = max_batch_size
             self.send_frequency = send_frequency
 
-            user_agent = "libhoney-py/" + VERSION
+            user_agent = "libhoney-py/" + VERSION + " (Tornado)"
```

Comment:
Ooo ooo, we could get at the Tornado version by including the following import with our other attempted tornado imports in transmission.py:

```python
from tornado import version as tornado_version
```

… and then include the version in our TornadoTransmission's user agent:

```suggestion
            user_agent = "libhoney-py/" + VERSION + " (Tornado/" + tornado_version +")"
```

I admit, I do not know if this is Distasteful Python or not.

Reply:
Oof … and what do you think about lowercase `tornado` because CONTAINS in queries is case-sensitive?